layers/lcm.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/google/gopacket"
+)
+
+// Lightweight Communications and Marshalling (LCM)
```

Comment:
Move this comment down to be the godoc comment for the LCM struct, so it shows up in Godoc?